flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/slotpool/DualKeyLinkedMap.java

```diff
@@ -37,20 +44,20 @@
 
 	private final LinkedHashMap<A, Tuple2<B, V>> aMap;
 
-	private final LinkedHashMap<B, A> bMap;
+	private final HashMap<B, A> bMap;
 
 	private transient Collection<V> values;
 
 	public DualKeyLinkedMap(int initialCapacity) {
```

Comment:
@azagrebin what do you think of making this class package private?
It looks like a common util class but no one other than `SlotPoolImpl` uses it.

Reply:
Makes sense to me